qi-lib/flow/core/compiler.rkt

```diff
@@ -4,9 +4,9 @@
 
 (require (for-syntax racket/base
                      syntax/parse)
-         "passes/pass-1000-qi0.rkt"
```

Comment:
RE: 3b53feccad4a41446cc34cc31597afda92969229, At risk of making your life even more complicated, if this is where we are going
I would prefer to avoid the intermediate commits that use the passes/ directory.
Git can follow the renames pretty well (see `-C` and `-M` to commands like
`blame` and `log`, to the extent that when I really care I end up using `-CCC
-M`). Still, avoiding clutter in the history will make future contributions
easier.